bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3Metrics.java

```diff
@@ -1,43 +0,0 @@
-/**
- * Copyright Pravega Authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.pravega.storage.extendeds3;
-
-import io.pravega.shared.MetricsNames;
-import io.pravega.shared.metrics.Counter;
-import io.pravega.shared.metrics.MetricsProvider;
-import io.pravega.shared.metrics.OpStatsLogger;
-import io.pravega.shared.metrics.StatsLogger;
-
-/**
- * Defines all Metrics used by the ExtendedS3Storage class.
- */
-final class ExtendedS3Metrics {
```

Comment:
Why are you deleting all these files? Is this because `ChunkedSegmentStorage` and `WriterOperation` classes already report metrics, so these can be removed? If that is the case, are we sure that we are not reporting anything here that could be interesting to port to our current metrics?

Reply:
Some of these and other new metrics are now handled by SLTS https://github.com/pravega/pravega/blob/master/segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorageMetrics.java.

This class was used in old code that is no longer needed.